server/routes/api.js

```diff
@@ -146,53 +146,28 @@ router.post('/register',
       
 });
 
-// Update the role and admin permissions for the member
```

Comment:
I think everything else is good, but we seem to have lost the role/admin/delete member handling in the back-end